autoload/neomake/signs.vim

```diff
@@ -269,8 +270,9 @@ function! neomake#signs#DefineHighlights() abort
         let ctermfg = neomake#utils#GetHighlight(fg_group, fg_attr)
         let guifg = neomake#utils#GetHighlight(fg_group, fg_attr.'#')
         " Ensure that we're not using SignColumn bg as fg (as with gotham
-        " colorscheme, issue https://github.com/neomake/neomake/pull/659).
-        if ctermfg == ctermbg && guifg == guibg
+        " colorscheme, issue https://github.com/neomake/neomake/pull/659)
+        " or that the sign color is different that the Normal bg.
+        if (ctermfg == ctermbg && guifg == guibg) || ctermfg == normalctermbg
```

Comment:
Not really sure, but it should maybe also check `&& guifg == normalguibg` (with `let normalguibg = neomake#utils#GetHighlight('Normal', 'bg#')` ?
Is that the case with your scheme?